qiskit_nature/problems/second_quantization/lattice/models/fermi_hubbard_model.py

```diff
@@ -95,24 +74,7 @@ def from_parameters(
             ValueError: If the hopping matrix is not square matrix,
```

Comment:
### 
```suggestion
            ValueError: If the coupling matrix is not square matrix,
```

Reply:
I've pushed all the changes above in the last commit. The method `uniform_parameters` has an argument `uniform_hopping` should that be changed too to `uniform_coupling`?

Reply:
I think this will look more proper for Ising

Reply:
is that a yes?(sorry)

Reply:
Sorry, yes it is a yes